client/my-sites/themes/thanks-modal.jsx

```diff
@@ -134,14 +122,18 @@ const ThanksModal = createReactClass( {
 		return (
 			<div>
 				<h1>
-					{ translate( 'Thanks for choosing {{br/}} %(themeName)s {{br/}} by %(themeAuthor)s', {
-						args: { themeName, themeAuthor },
+					{ translate( 'Thanks for choosing {{br/}} %(themeName)s', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Thanks for choosing %1$s by %2$s.' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/change/theme-switch-flow)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).